src/administration/from_scratch.md

```diff
@@ -1,21 +1,82 @@
 # From Scratch
 
-These instructions are written for Ubuntu 20.04 / Ubuntu 22.04.
+These instructions are written for Ubuntu 20.04 / Ubuntu 22.04. They are particularly useful when you'd like to setup a Lemmy container (e.g. LXC on Proxmox) and cannot use Docker.
+
+Lemmy is built from source in this guide, so this may take a while, especially on slow devices. For example, Lemmy v0.18.5 takes around 7 minutes to build on a quad core VPS.
+
+Installing and configuring Lemmy using this guide takes about 60-90 minutes. You might need to make yourself a fresh cup of coffee before you start.
+
+While is not an officially supported installation method for Lemmy, it is not that hard. Maybe someone will eventually create an Ansible script for this as well.
 
 ## Installation
 
-### Lemmy Backend
+### Database
+
+For Ubuntu 20.04 the shipped PostgreSQL version is 12 which is not supported by Lemmy. So let's set up a newer one.
+The most recent stable version of PostgreSQL is 16 at the time of writing this guide.
+
+#### Install dependencies
+
+```
+sudo apt install wget ca-certificates
+sudo apt install pkg-config libssl-dev libpq-dev postgresql
+wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo apt-key add -
+sudo sh -c 'echo "deb http://apt.postgresql.org/pub/repos/apt/ $(lsb_release -cs)-pgdg main" >> /etc/apt/sources.list.d/pgdg.list'
+sudo apt update
+sudo apt install pkg-config libssl-dev libpq-dev postgresql
+```
+
+#### Setup Lemmy database
+
+Replace db-passwd with a unique password of your choice in the commands below.
+
+```
+sudo -iu postgres psql -c "CREATE USER lemmy WITH PASSWORD 'db-passwd';"
+sudo -iu postgres psql -c "CREATE DATABASE lemmy WITH OWNER lemmy;"
+```
+
+If you're migrating from an older version of Lemmy, the following might be required.
+
+```
+sudo -iu postgres psql -c "ALTER USER lemmy WITH SUPERUSER;"
+```
+
+Tune your PostgreSQL settings to match your hardware via [this guide](https://pgtune.leopard.in.ua/#/)
+
+#### Setup md5 auth
+
+Lemmy currently only supports only non-SSL connections to databases. More info [here](https://github.com/LemmyNet/lemmy/issues/3007).
```

Comment:
That issue is already resolved.

Reply:
This was present in the original guide. I will review and update.